common/web/keyboard-processor/src/keyboards/defaultLayouts.ts

```diff
@@ -19,7 +19,26 @@ export { ButtonClasses };
 import Codes from "../text/codes.js";
 import type Keyboard from "./keyboard.js";
 
-export type KLS = {[layerName: string]: string[]};
+export interface EncodedVisualKeyboard {
+  /** Represents CSS font styling to use for VisualKeyboard text */
+  F: string;
+  /** Should there be a 102nd key? */
+  K102?: boolean,
+  /**
+   * Keyboard Layer Specification: an object-based map of layer name to the keycaps for its
+   * 65 keys.  The 65 keys are ordered from left to right, then top to bottom.
```

Comment:
Why 65 (keys)?

Reply:
Took some digging, but I found it.

This array is precisely 65 entries long and maps each index of the 65 to its represented hardware key.  Note that some are reserved - that's the role of `K_*`.

https://github.com/keymanapp/keyman/blob/a024274a62e1f3d0f197af2b67a90a757404f348/common/web/keyboard-processor/src/keyboards/defaultLayouts.ts#L51-L59

Reply:
In regard to how they're linked:

Web has a built-in default desktop layout, listing the key IDs at each position.
https://github.com/keymanapp/keyman/blob/a024274a62e1f3d0f197af2b67a90a757404f348/common/web/keyboard-processor/src/keyboards/defaultLayouts.ts#L551-L567

When iterating through this default layout, we take the key ID and look up its position among the 65.
https://github.com/keymanapp/keyman/blob/a024274a62e1f3d0f197af2b67a90a757404f348/common/web/keyboard-processor/src/keyboards/defaultLayouts.ts#L234-L247